app/routes/events/components/EventEditor/index.js

```diff
@@ -56,7 +56,8 @@ type Props = {
   setCoverPhoto: (number, string) => void,
   deleteEvent: (eventId: ID) => Promise<*>,
   submitting: boolean,
-  pristine: boolean
+  pristine: boolean,
+  initialized: boolean
```

Comment:
Where does this prop come from? And what does it do?

Reply:
I was going to make a comment about, I knew I forgot something. The reason for this prop is an [issue](https://github.com/redux-form/redux-form/issues/621) in redux form that causes all fields to be initially rendered with an empty string as value, due to the form not being initialized. Due to the editorstate being immutable, the editorfield does not update once it is passed the correct initialvalue on the second render. This props "solves" the issue by enabling the editorfield to only render once the form has been initialized. The initialized prop is passed to the form by the redux-form HoC, see [this](https://redux-form.com/8.3.0/docs/api/props.md/) page for details. The "solution" is a hack, yet i could find no better way to solve this.

Reply:
Ah, I see, it's not our bad code :1st_place_medal:. You should add a comment in the `EditorField` component explaining this then.

Reply:
Will do